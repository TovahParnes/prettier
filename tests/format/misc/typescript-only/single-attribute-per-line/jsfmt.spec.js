run_spec(__dirname, ["typescript"], { singleAttributePerLine: true });
