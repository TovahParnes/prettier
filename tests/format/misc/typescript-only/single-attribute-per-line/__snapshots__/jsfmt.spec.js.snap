// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`single-attribute-per-line.ts - {"singleAttributePerLine":true} format 1`] = `
====================================options=====================================
parsers: ["typescript"]
printWidth: 80
singleAttributePerLine: true
                                                                                | printWidth
=====================================input======================================
function doX(
  editor, 
  triggerPosition, 
  re
  ) {}

class X {
  constructor(editor, triggerPosition, suggestions){}
}

=====================================output=====================================
function doX(editor, triggerPosition, re) {}

class X {
  constructor(
    editor,
    triggerPosition,
    suggestions
  ) {}
}

================================================================================
`;
